linux_os/guide/system/software/integrity/crypto/var_ssh_client_rekey_limit_time.var

```diff
@@ -0,0 +1,14 @@
+documentation_complete: true
+
+title: 'SSH client RekeyLimit - size'
+
+description: 'Specify the size component of the rekey limit.'
+
+type: string
+
+operator: equals
+
+options:
+    ssh_client_default: "none"
```

Comment:
This is confusing. `ssh_client_default` should just be the default if we really need to maintain the ssh client default.... and from a security standpoint, I don't think that we should maintain the default. Also, can this variable be interactive?